lib/client/ca_export.go

```diff
@@ -423,9 +392,33 @@ func hostCAFormat(ca types.CertAuthority, keyBytes []byte, client authclient.Cli
 	})
 }
 
-func exportAuthForIntegration(ctx context.Context, client authclient.ClientI, req ExportAuthoritiesRequest) ([]*ExportedAuthority, error) {
-	switch req.AuthType {
-	case "github":
+// IsIntegrationAuthorityType returns true if provided type is an integration CA
+// type.
+func IsIntegrationAuthorityType(exportType string) bool {
+	return exportType == types.IntegrationSubKindGitHub
+}
+
+// ExportIntegrationAuthoritiesRequest has the required fields to create an
+// export authorities request for integrations.
+type ExportIntegrationAuthoritiesRequest struct {
+	// Type is the integration type.
+	Type string
+	// MatchFingerprint filters authorities using provided fingerprint if
```

Comment:
What is the format for the fingerprint? What data is fingerprinted? How does one go about getting a fingerprint for this?

Reply:
i have added an example in the new github section on how this option can be used.